packages/web3-core-method/src/index.js

```diff
@@ -361,7 +362,14 @@ Method.prototype._confirmTransaction = function (defer, result, payload) {
                         if(receipt) {
                             receipt = JSON.stringify(receipt, null, 2);
                         }
-                        utils._fireError(new Error("Transaction ran out of gas. Please provide more gas:\n"+ receipt), defer.eventEmitter, defer.reject);
+                        if (receipt.status === '0x0') {
```

Comment:
Thanks for that, but could you add the status parsing filed to the web3-core-helpers formatters, so that it returns `status: true` or `false`, instead of the 0x1, and then change this code here to check for that.